app/src/main/java/com/mealsmadeeasy/ui/splash/SplashActivity.kt

```diff
@@ -2,14 +2,13 @@ package com.mealsmadeeasy.ui.splash
 
 import android.os.Bundle
 import com.mealsmadeeasy.ui.BaseActivity
-import com.mealsmadeeasy.ui.home.HomeActivity
+import com.mealsmadeeasy.ui.login.LoginActivity
 
 class SplashActivity : BaseActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        startActivity(HomeActivity.newIntent(this))
+        startActivity(LoginActivity.newIntent(this))
```

Comment:
This should still go to the HomeActivity if the user was already logged in. So it probably needs to be
```
if (FirebaseAuth.getInstance().getCurrentUser() == null) {
    // Launch LoginActivity
} else {
    // Launch HomeActivity
}